src/@providers/docker/services/build.ts

```diff
@@ -31,9 +32,16 @@ export class DockerBuildService extends CrudResourceService<'dockerBuild', Docke
       args.push('-f', inputs.dockerfile);
     }
 
-    args.push(inputs.context);
+    const context = path.isAbsolute(inputs.context)
+      ? inputs.context
+      : path.join(path.dirname(inputs.component_source), inputs.context);
+    args.push('./');
+    // args.push(context);
```

Comment:
This looks like it was debugging code. The context is the one you want, right?

Reply:
This did need to get removed and I switched it so the cwd is the context instead of giving docker the context.